doc/users/prev_whats_new/whats_new_3.1.0.rst

```diff
@@ -338,7 +338,7 @@ MouseEvent button attribute is now an IntEnum
 The :attr:`button` attribute of `~.MouseEvent` instances can take the values
 None, 1 (left button), 2 (middle button), 3 (right button), "up" (scroll), and
 "down" (scroll).  For better legibility, the 1, 2, and 3 values are now
-represented using the `IntEnum` class `matplotlib.backend_bases.MouseButton`,
+represented using the `.IntEnum` class `matplotlib.backend_bases.MouseButton`,
```

Comment:
does .IntEnum help? It's not part of the library.

Reply:
ah, it doesn't, reverted